index.html

```diff
@@ -63,5 +63,169 @@ <h2>I’m a software developer</h2>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+            </ul>
+            <br>
+            <button>See Project</button>
+            <br>
+        </div>
+        </div>
+        <div class="grid">
+        <div class="grid1"></div>
+        <div class="grid2">
+            <br>
+            <h2>Multi-Post Stories <br> Gain+Glory</h2>
+            <br>
+            <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+            </ul>
+            <br>
+            <button>See Project</button>
+            <br>
+        </div>
+        </div>
+        <div class="grid">
+            <div class="grid1"></div>
+            <div class="grid2">
+                 <br>
+                <h2>Multi-Post Stories <br> Gain+Glory</h2>
+                <br>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>Javascript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <button>See Project</button>
+                <br>
+            </div>
+        </div>
+        <div class="grid">
+            <div class="grid1"></div>
+            <div class="grid2">
+                <br>
+                <h2>Multi-Post Stories <br> Gain+Glory</h2>
+                <br>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>Javascript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <button>See Project</button>
+                <br>
+            </div>
+        </div>
+        <div class="grid">
+            <div class="grid1"></div>
+            <div class="grid2">
+                <br>
+                <h2>Multi-Post Stories <br> Gain+Glory</h2>
+                <br>
+                 <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>Javascript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <button>See Project</button>
+                <br>
+            </div>
+        </div>
+        <div class="grid">
+            <div class="grid1"></div>
+            <div class="grid2">
+                <br>
+                <h2>Multi-Post Stories <br> Gain+Glory</h2>
+                <br>
+                 <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>Javascript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <button>See Project</button>
+                <br>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="abt-main">
+            <br>
+            <h1>About me</h1>
```

Comment:
- [ ] As requested by the previous reviewer for schematic reasons, only one `h1` element should be in all your document. Since you have it in the intro, please consider changing all the other `h1` elements. The main reason for this is that search engines use the `h1` element to understand the primary topic of the page. If there are multiple `h1` elements on a page, it can confuse search engines and make it more difficult for them to accurately determine the main topic of the page.